### PR TITLE
Added my project

### PR DESCRIPTION
**Pull request by CearceafAlb**
This pull request adds my project documentation.

Signed-off-by: Leonte Robert robert.leonte@stud.fils.upb.ro

### CODE REVIEW

File: docs/project/2024/robert.leonte/index.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+| [Device](link://to/device) | This is used ... | [price](link://to/store) |
+
+```
+
+-->
+
+| Device | Usage | Price |
+|--------|--------|-------|
+| [Rapspberry Pi Pico W](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html) | The microcontroller | [35 RON](https://www.optimusdigital.ro/en/raspberry-pi-boards/12394-raspberry-pi-pico-w.html) |
+| [Breadboard(taken from a kit)] | The base | [54 EUR(the entire kit)](https://www.amazon.de/gp/product/B01II76PDM/ref=ox_sc_act_title_1?smid=AZF7WYXU5ZANW&psc=1) |
```
Comment (alexandruradovici) [inline docs/project/2024/robert.leonte/index.md:56]: No square brackets if here is no link. Please do add links to your components.

File: docs/project/2024/robert.leonte/index.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+| [Thumbstick] | Bi-axial module for movement and aiming | [5,35 RON](https://www.optimusdigital.ro/ro/senzori-senzori-de-atingere/742-modul-joystick-ps2-biaxial-negru-cu-5-pini.html) |
+| [Thumbstick(taken from a kit)] | Bi-axial module for movement and aiming | [54 EUR(the entire kit)](https://www.amazon.de/gp/product/B01II76PDM/ref=ox_sc_act_title_1?smid=AZF7WYXU5ZANW&psc=1) |
+
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [st7789](https://github.com/almindor/st7789) | Display driver for ST7789 | Used for the display for the Pico Explorer Base |
+| [embedded-graphics](https://github.com/embedded-graphics/embedded-graphics) | 2D graphics library | Used for drawing to the display |
```
Comment (alexandruradovici) [inline docs/project/2024/robert.leonte/index.md:67]: Do you need these libraries?

File: docs/project/2024/robert.leonte/index.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+## Links
+
+<!-- Add a few links that inspired you and that you think you will use for your project -->
+
+1. [link](https://example.com)
+2. [link](https://example3.com)
```
Comment (alexandruradovici) [inline docs/project/2024/robert.leonte/index.md:74]: Please add links for your project.

File: docs/project/2024/robert.leonte/index.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [embedded-graphics](https://github.com/embedded-graphics/embedded-graphics) | 2D graphics library | Used for drawing to the display(and a small image, a logo maybe, when the controller is plugged in for the first time) |
+| [GameInput](https://learn.microsoft.com/en-us/gaming/gdk/_content/gc/input/overviews/input-overview) | A new input API from Microsoft that aims to bring XInput, DirectInput, etc under a single roof | Used for handling the buttons and the joysticks |
+
+## Links
+
+<!-- Add a few links that inspired you and that you think you will use for your project -->
+
+1. [link](https://learn.microsoft.com/en-us/gaming/gdk/_content/gc/input/overviews/input-overview)
```
Comment (alexandruradovici) [inline docs/project/2024/robert.leonte/index.md:73]: Please add a title to the link.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]
Please fill the pull request description following the template.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]
Please fill the pull request description following the template.

**Review by CearceafAlb — COMMENTED** [review]
Made the changes

**Comment by CearceafAlb** [comment]
Apologies, I have added it now. Hope everything is ok.

File: docs/project/2024/robert.leonte/index.md (hunk 4 above)
Comment (alexandruradovici) [reply docs/project/2024/robert.leonte/index.md:73]: Not resolved.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Review by CearceafAlb — COMMENTED** [review]
Added the title to the link.

**Review by alexandruradovici — APPROVED** [review]

**Review by irina-nita — APPROVED** [review]

**Pull request merged**